charts/sourcegraph/CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Use `**BREAKING**:` to denote a breaking change
 
 ## Unreleased
 
+
+## 3.43.0
+
+- Sourcegraph 3.43.0 is now available
```

Comment:
guessing this will be updated once all the posts are alive?

Reply:
I don't think there are any changes, which is why the content is empty.